user_guide/src/coming_from_pandas.md

```diff
@@ -1,44 +1,155 @@
+
+Here we set out the key points that anyone who has experience with `Pandas` and wants to
+try `Polars` should know. We include both differences in the concepts the libraries are
+built on and differences in how you should write `Polars` code compared to `Pandas`
+code.
+
+## Differences in concepts between `Polars` and `Pandas`
+
+### `Polars` does not have an index
+
+`Pandas` gives a label to each row with an index. `Polars` does not use an index and
+each row is indexed by its integer position in the table.
+
+Indexes are not needed! Not having them makes things easier - convince us otherwise!
+
+For more detail on how you select data in `Polars` see the [indexing](indexing.md)
+section.
+
+### `Polars` uses Apache Arrow arrays to represent data in memory while `Pandas` uses `Numpy` arrays
+
+`Polars` represents data in memory with Arrow arrays while `Pandas` represents data in
+memory in `Numpy` arrays. Apache Arrow is an emerging standard for in-memory columnar
+analytics that can accelerate data load times, reduce memory usage and accelerate
+calculations.
+
+`Polars` can convert data to `Numpy` format with the `to_numpy` method.
+
+### `Polars` has more support for parallel operations than in `Pandas`
+
+`Polars` exploits the strong support for concurrency in Rust to run many operations in
+parallel. While some operations in `Pandas` are multi-threaded the core of the library
+is single-threaded and an additional library such as `Dask` must be used to parallelise
+operations.
+
+### `Polars` can lazily evaluate queries and apply query optimization
+
+Eager evaluation is where code is evaluated as soon as you run the code. Lazy evaluation
+is where running a line of code means that the underlying logic is added to a query plan
+rather than being evaluated.
+
+`Polars` supports eager evaluation and lazy evaluation whereas `Pandas` only supports
+eager evaluation. The lazy evaluation mode is powerful because `Polars` carries out
+automatic query optimization where it examines the query plan and looks for ways to
+accelerate the query or reduce memory usage.
+
+`Dask` also supports lazy evaluation where it generates a query plan. However, `Dask`
+does not carry out query optimization on the query plan.
+
+## Key syntax differences
 
 Users coming from `Pandas` generally need to know one thing...
 
 ```
 polars != pandas
 ```
 
-If your `Polars` code looks like it could be `Pandas` code, it might run, but it likely runs slower than it should.
+If your `Polars` code looks like it could be `Pandas` code, it might run, but it likely
+runs slower than it should.
 
 Let's go through some typical `Pandas` code and see how we might write that in `Polars`.
 
-## Column assignment
+### Selecting data
+
+As there is no index in `Polars` there is no `.loc` or `iloc` method in `Polars` - and
+there is also no `SettingWithCopyWarning` in `Polars`.
+
+To learn more about how you select data in `Polars` see the [indexing](indexing.md)
```

Comment:
I would want to add here that in idiomatic polars use, accessing data by index is much less prevalent as we use the expression API to access our data.